code/src/components/TodoItem.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+
+import { useDispatch, useSelector } from "react-redux";
+import { todos } from "../reducers/todos.js"
+import { CustomCheckbox } from "lib/CustomCheckbox.js";
+import styled from 'styled-components'
+import { Button } from '../lib/Button'
+
+
+export const TodoItem = ({ itemIndex }) => {
+    const item = useSelector((store) => store.todos.list.items[itemIndex]);
+
+    const dispatch = useDispatch();
+
+    const handleOnChange = (e) => {
+
+        dispatch (
+            todos.actions.setDone({
+                itemIndex: itemIndex,
+                done: !item.done,
+            })
+        );
+    };
+
+    const onRemoveClicked = (e) => {
+
+        dispatch (
+            todos.actions.removeTodo({
+                itemIndex,
+            })
+        );
+    };
+
+
+    const Itemtext = styled.text`
```

Comment:
`ItemText` is a more standard way of writing something that is two words combined